donkeycar/parts/keras.py

```diff
@@ -140,6 +140,40 @@ def train(self,
             workers=1,
             use_multiprocessing=False
         )
+            
+        try:
```

Comment:
We used to have a flag that enabled/disabled that part which I think is useful because the plot actually blocks the call. So if you run training in a batch it doesn't return until you click the matplot image away, for that we need to be able to suppress that.

Reply:
Right. I have temporarily commented out `plt.show()`. Let's add it back later.

Reply:
Ah I see you're right.